docs/fast_data/configuration/projection_storer.md

```diff
@@ -954,6 +954,23 @@ projections:
 </p>
 </details>
 
+## Runtime Management
```

Comment:
This part is overlapping with the changes introduced here, that are waiting to be merged. How should we integrate them?

Reply:
I'll merge the PR #1266 onto these changes to include latter edits to the service documentation

Reply:
Done